scripts/zones/FeiYin/mobs/Clockwork_Pod.lua

```diff
@@ -17,5 +17,5 @@ function onMobDeath(mob, player, isKiller)
 end
 
 function onMobDespawn(mob)
-    dsp.mob.phOnDespawn(mob, ID.mob.DONNERGUGI_PH, 10, 3600) -- 1 hour
+    dsp.mob.phOnDespawn(mob, ID.mob.MIND_HOARDER_PH, 10, (math.random(5400, 32400))) -- 1.5 to 9 hours
```

Comment:
```lua
    dsp.mob.phOnDespawn(mob, ID.mob.MIND_HOARDER_PH, 10, math.random(5400, 32400)) -- 1.5 to 9 hours
```
Don't need extra parentheses

Reply:
Need 3 at the end to close whole function. Travis already errored on it with 2 at end

Reply:
because there was an extra leading one in front of the math random.

Reply:
@KnowOne134 I'll make thsi really simple for you to visualize:

thing(stuff inside it)

thing(thing(stuff inside it))

thing(thing(thing(stuff inside it)))

thing(thing(stuff inside (it)))

(it) was wrong.